internal/buffer/save.go

```diff
@@ -233,7 +272,13 @@ func (b *Buffer) saveToFile(filename string, withSudo bool, autoSave bool) error
 		}
 	}
 
-	if err = b.safeWrite(absFilename, withSudo); err != nil {
+	errorChan := make(chan error, 1)
```

Comment:
This channel doesn't need to be buffered? It is guaranteed that the main goroutine waits on it when the save goroutine sends the error to it?